components/useExternalInput.module.scss

```diff
@@ -0,0 +1,28 @@
+.root::after,
```

Comment:
The style is copied from GitHub's own overlay when dragging files over the "Code" tab in a repo

<table>
<tr>
	<th>GitHub
	<th>npmgraph
<tr>
	<td><img width="670" alt="Screenshot 7" src="https://github.com/user-attachments/assets/5790942d-f535-4b41-a1f0-bdd8625b7f8f">
	<td><img width="670" alt="Screenshot 6" src="https://github.com/user-attachments/assets/5bb187bc-7dc8-40b6-9979-fb787cce15bd">
</table>